docs/guides/chip_tool_guide.md

```diff
@@ -298,8 +308,9 @@ In this command:
 
 #### Forgetting the already-commissioned device
 
-The following command removes the device with the given node ID from the list of
-commissioned Matter devices:
+In case when you need to retest the commissioning steps, the following command
+removes the device with the given node ID from the list of commissioned Matter
```

Comment:
```suggestion
In case commissioning needs to be retested, the following command
removes the device with the given node ID from the list of commissioned Matter
```